doc/installation.md

```diff
@@ -0,0 +1,98 @@
+## Introduction
+
+The workflow of the installation is customized via the [control
+file](https://github.com/yast/yast-installation/blob/master/doc/control-file.md), 
+and below we have the principal steps concerning network configuration:
+
+  - Linuxrc
+  - inst_install_inf (What is still needed)
+  - inst_dhcp
+  - manual_configuration (openSUSE only in Addons)
```

Comment:
I think in SLE it is in registration, so in both you have chance to set it, not?